espnet/nets/pytorch_backend/e2e_tts_tacotron2.py

```diff
@@ -23,14 +23,18 @@
 
 
 class GuidedAttentionLoss(torch.nn.Module):
-    """Guided attention loss function
+    """Guided attention loss function module.
 
-    Reference:
-        Efficiently Trainable Text-to-Speech System Based on Deep Convolutional Networks with Guided Attention
-        (https://arxiv.org/abs/1710.08969)
+    This module calculates the guided attention loss described in `Efficiently Trainable Text-to-Speech System Based
+    on Deep Convolutional Networks with Guided Attention`_, which forces the attention to be diagonal.
+
+    Args:
+        sigma (float, optional): Standard deviation to control how close attention to a diagonal.
+        reset_always (bool, optional): Whether to always reset masks.
+
+    .. _`Efficiently Trainable Text-to-Speech System Based on Deep Convolutional Networks with Guided Attention`:
```

Comment:
use `References` section directive? http://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#confval-napoleon_use_admonition_for_references

Reply:
What do you mean?
use `napoleon_use_admonition_for_references` option?
e.g.
```python
extensions = [
    "sphinx.ext.autodoc",
    'sphinx.ext.napoleon',
    'sphinx.ext.viewcode',
    "sphinx.ext.mathjax",
    "sphinxarg.ext",
]

# napoleon setting
napoleon_use_admonition_for_references = True
````

Reply:
Sorry, I misunderstood what the options is.